### PR TITLE
Added other examples of project management software

### PR DESCRIPTION
**Pull request by Loco1982**
We don't use only clickup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -194,7 +194,7 @@ Strive for solving problems in the simplest way possible. To achieve this, you f
 
 ### Development process/workflow
 - There is a weekly call with every member of the team to set the most important goals of the week, as well as a daily call to focus on the problems of the daily tasks.
-- When working on projects, the specific tasks to tackle are written in clickup.
+- When working on projects, the specific tasks to tackle are written in tickets on a project management software (i.e. Clickup, Trello, Jira, or Github Projects).
```
Comment (matias-gonz) [inline README.md:197]:
```suggestion
- When working on projects, the specific tasks to tackle are written in tickets on some project management software (i.e. Clickup, Trello, Jira, or Github Projects).
```

**Review by matias-gonz — APPROVED** [review]
I left a suggestion but it is not a big deal

**Review by tomasdema — CHANGES_REQUESTED** [review]
I would say our standard tool is Github Projects but we have some legacy tools like clickup, trello etc. If we have a new project we should use github projects.

**Comment by Loco1982** [comment]
> I would say our standard tool is Github Projects but we have some legacy tools like clickup, trello etc. If we have a new project we should use github projects.

You're right, made the changes to the text!

**Review by tomasdema — APPROVED** [review]

**Pull request merged**